service/chartconfig/v1/appr/spec.go

```diff
@@ -0,0 +1,5 @@
+package appr
+
+type Package struct {
```

Comment:
Just for personal interest: Why is this named `Package`? I've only seen `Application` in quay.io so far, is this CNR terminology?

Reply:
Exactly, `Package` is a concept from CNR, see for instance https://github.com/app-registry/appr/blob/master/appr/api/impl/registry.py#L198